conf/application.conf

```diff
@@ -250,3 +252,22 @@ hmrc-timeout-dialog {
   enableSynchroniseTabs = true
   defaultCountdownInSeconds = 120
 }
+
+sca-wrapper {
+    future-timeout = 5
+    host = "http://localhost:9514" # used in feedback url
+    feedback-service-name = "CYSP/pension"
+    exit-survey-origin = "CYSP"
+    accessibility-statement {
+      referrer.url = "/check-your-state-pension"
+      redirect.url = "check-your-state-pension"
+    }
+    service-name {
+      messages-key = "Check or update your medical insurance benefit"
```

Comment:
Should this be a message key and is this value correct?

Reply:
It has been removed.